packages/common/src/types/index.ts

```diff
@@ -52,78 +51,3 @@ export type Store = {
   id: string;
   name: string;
 };
-
-export interface InvoiceLine extends InvoiceLineNode, DomainObject {
```

Comment:
So all of this moved to `invoices` package.. Think the other ones should go in their packages also - i.e. items/name into system. Stock lines.. uhh.. system also?

Reply:
hmm, yeah. stock lines are shared across a number of areas.. might be best to have them in system